tests/test_plugins.py

```diff
@@ -1,3 +1,4 @@
+### Are these needed? Some of those seem already present within other test modules
```

Comment:
that was long ago, kind of futures (https://python-future.org/imports.html)

No more necessary since we do not support python2 anymore.